infrastructure/idam-preview.tfvars

```diff
@@ -1,3 +1,3 @@
 ga_tracking_id = "UA-122164129-3"
-idam_api_url_override = "https://idam-api-idam-aat.service.core-compute-idam-aat.internal"
+idam_api_url_override = "https://idam-api.aat.platform.hmcts.net"
```

Comment:
Could we get this from the environment variable so we don't need to change the value in the future?